tests/test_utils.py

```diff
@@ -0,0 +1,40 @@
+import os
+import numpy as np
+from tempfile import TemporaryDirectory
+from autowebcompat import utils
+
+
+def test_get_bugs():
+    Object = utils.get_bugs()
+    assert(isinstance(Object, list))
+
+
+def test_mkdir():
+    d = TemporaryDirectory()
+    direc_path = os.path.join(d.name + "\\test")
+    utils.mkdir(direc_path)
+    assert(os.path.isdir(direc_path))
```

Comment:
Second part from https://github.com/marco-c/autowebcompat/pull/63#discussion_r168962349

Reply:
@marco-c  Instead of writing a test for it , would it not make more sense to change the mkdir function to check if a directory with the same name exists and not make it if it does? since the function the doesnt return any error message it is pretty hard to assert on anything . If some error message were returned i could assert on the same .

Reply:
You don't need to add an assertion, the assertion is implied in the fact that `utils.mkdir` doesn't throw an exception.